app/views/photos/_photo.html.erb

```diff
@@ -6,19 +6,21 @@
       <%= link_to photo.owner.username, user_path(photo.owner.username), class: "text-dark" %>
     </h2>
 
-    <div>
-      <%= link_to edit_photo_path(photo), class: "btn btn-link btn-sm text-muted" do %>
-        <i class="fas fa-edit fa-fw"></i>
-      <% end %>
+    <% if current_user == photo.owner%>
```

Comment:
It's better to replace this by:
Might want to add this here:
```rb
<% if policy(photo).edit? %>
```
so we have the policy logic in the same place